src/basic/proc-cmdline.c

```diff
@@ -39,6 +39,39 @@ int proc_cmdline(char **ret) {
                 return read_one_line_file("/proc/cmdline", ret);
 }
 
+
+/* note; similar function is also in libshared, but libbasic cant't depend
+* on libshared */
+static bool is_efi_secure_boot() {
```

Comment:
Not sure if this function should go in this file; but it is only used here.

It's also duplicated code from https://github.com/systemd/systemd/blob/master/src/shared/efi-loader.c